src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLifeCycleManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
```

Comment:
Did you want to use a file-scoped namespace here as you've done elsewhere?

Reply:
I think we had discussed a discussion about this on another issue and we decided we should [do it all at once](https://github.com/dotnet/razor-tooling/issues/6895).